pkg/controllers/node/node_controller.go

```diff
@@ -159,139 +104,112 @@ func (c *NodeController) Run(threadiness int, reconcilerPeriod string, stopCh ch
 	log.Debug("Finished syncing with Kubernetes API (Nodes)")
```

Comment:
I think the loop above is a simple tight loop; probably worth throwing a 100ms sleep in there.